private/buf/buflsp/file.go

```diff
@@ -45,7 +45,13 @@ import (
 	"google.golang.org/protobuf/reflect/protoreflect"
 )
 
-const descriptorPath = "google/protobuf/descriptor.proto"
+const (
+	descriptorPath = "google/protobuf/descriptor.proto"
+)
+
+var (
+	errFileInfoNoLocalPath = errors.New("file info has no local path")
```

Comment:
Is this error printed for a user? Even if not, I don't actually know what it means - what does it mean for a file info to not have a path? This should be clearer

Reply:
It is printed in logs, but not directly exposed to the user in-line. I can clarify this error, however it is mostly for logging and debugging.